app/static/tests/e2e/code.etest.ts

```diff
@@ -20,6 +20,13 @@ b     <-  user(3.0)
 test.describe("Code Tab tests", () => {
     const { timeout } = PlaywrightConfig;
 
+    const writeCode = async (page: Page, code: string) => {
+        await page.press(".monaco-editor textarea", "Control+A");
+        await page.press(".monaco-editor textarea", "Delete");
+        await page.fill(".monaco-editor textarea", "");
```

Comment:
MacBook does not recognise Control+A, so tests failed locally. Adding this line works for me locally as well as Buildkite.